### DISCUSSION TITLE
Cara Buka

### DISCUSSION THREAD

**Discussion by andsoe12** [lintangtimur/ovoid#42] [General]
Maaf bang mau nanya ini cara bukanya gimana ya? sebelumnya terimakasi.

**Comment by lintangtimur** [comment]
cara buka apa ya?

**Reply by andsoe12** [reply]
Ini mas, ijin nanya, saya masih pemula, ini cara buka web nya gimana ya? Terimakasih sebelumnya

**Reply by lintangtimur** [reply]
ermm, secara default ini bukan tampilan web. Ini library backend untuk transaksi layaknya aplikasi ovo.

**Reply by andsoe12** [reply]
Oalah saya kira ini semacam kaya codeigniter gitu, saya lg ada tubes untuk membuat aplikasi web ovo. Btw terimakasih mas kalau begitu.
